### ISSUE TITLE
Unexpected token (Note that you need @rollup/plugin-json to import JSON files)

### ISSUE THREAD

**Issue by brandonros** [preactjs/wmr#251]
To reproduce:

index.js:

```
import * as foo from 'browserify-cipher'
console.log(foo)
```

index.html:

```
<!DOCTYPE html>
<html>
  <head>
  </head>
  <body>
    <script type="module" src="/index.js"></script>
  </body>
</html>

```

`wmr build`

I can't figure out how to turn on "verbosity" to get a full stack/more logs. Might be something to consider?

**Comment by developit** [comment]
You can enable debug stacks using `DEBUG=1 npm run build`, though they're not always useful. 

FWIW loading Node crypto libraries in the browser generally produces very large bundles, even if you can get them to be bundled. Maybe give the Web Crypto API a try?

**Comment by brandonros** [comment]
```
$ DEBUG=1 ./node_modules/.bin/wmr build
Error: Unexpected token (Note that you need @rollup/plugin-json to import JSON files)
    at vr (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:85597)
    at Lo.error (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:166363)
    at To (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:164515)
    at Lo.setSource (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:171648)
    at lu.addModuleSource (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:316003)
    at async lu.fetchModule (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:317072)
    at async Promise.all (index 2)
    at async lu.fetchStaticDependencies (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:317573)
    at async Promise.all (index 0)
    at async lu.fetchModule (/Users/brandonros/Desktop/crypto/node_modules/wmr/wmr.cjs:2:317106)
```

**Comment by marvinhagemeister** [comment]
This is fixed with #480  🎉
